jazz-ui/src/assets/data/jazz-installer-vars.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
Why was this file copied here? The UI build references the actual source file for configuring UI so this file is just defunct per my understanding.